src/fparser/api.py

```diff
@@ -224,7 +226,8 @@ def parse(input, isfree=None, isstrict=None, include_dirs=None,
     get_reader
     """
     from .parsefortran import FortranParser
-    reader = get_reader(input, isfree, isstrict, include_dirs, source_only)
+    reader = get_reader(input, isfree, isstrict, include_dirs, source_only,
+                        ignore_comments=ignore_comments)
```

Comment:
Do we want to default to ignoring comments, now that we support them?

Reply:
I think so given that that was effectively the behaviour before.